bundles/org.openhab.binding.pihole/src/main/java/org/openhab/binding/pihole/internal/PiHoleHandler.java

```diff
@@ -0,0 +1,292 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pihole.internal;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.openhab.binding.pihole.internal.PiHoleBindingConstants.Channels.ADS_BLOCKED_TODAY_CHANNEL;
```

Comment:
My IDE replaces this by `import static org.openhab.binding.pihole.internal.PiHoleBindingConstants.Channels.*`
Also the lines above

Reply:
I think star imports are not preferred

Reply:
I think it is [...](https://github.com/search?q=repo%3Aopenhab%2Fopenhab-addons%20Constants.*&type=code)

Reply:
Only static imports though.

Reply:
From how many imports I should use star?

![obraz](https://github.com/openhab/openhab-addons/assets/2962846/9c78b944-007c-400f-9353-1129ef70a00c)

Reply:
Starting at 2 we use to have star.

Reply:
Note, I do not intend to bother you, but I kind a know quite well what will be a roadblocker down the road when finalizing your PR for inclusion in codebase.